packages/fetchai/skills/echo/tests/test_handlers.py

```diff
@@ -61,6 +61,10 @@ def setup_class(cls):
             ),
         )
 
+    def teardown(self) -> None:
```

Comment:
BaseSkillTestCase already has a teardown, why are you overriding it?

Reply:
I introduced that only at the end, removed this now